src/display.c

```diff
@@ -574,8 +574,10 @@ get_input(int prompt_position, struct key *key)
 			delay = watch_periodic(opt_refresh_interval);
 			foreach_displayed_view (view, i) {
 				if (view_can_refresh(view) &&
-				    watch_dirty(&view->watch))
+				    watch_dirty(&view->watch)){
+					load_refs(true);
```

Comment:
It would be nice to ensure `load_refs` is only called once if a refresh is needed. For example by adding a `bool refs_refreshed` outside the loop to guard.

Also please put a space before the `{`